gr-blocks/grc/xmlrpc_client.block.yml

```diff
@@ -14,18 +14,18 @@ parameters:
 -   id: callback
     label: Callback
     dtype: raw
```

Comment:
The Callback's `dtype` can be a string

Reply:
Or it can continue being a raw, but that would typically require quotes

Reply:
If you make a callback‘s dtype a string it will put it to the  generated python code with quotes around The actual function call which throws an error (rightfully so).

Reply:
I thought raw would indiscriminately pass along whatever text you gave it. Could this possibly be a problem with the Python code generation for raw type? Odd that it used to work fine.

Reply:
`-   id: callback`
    `label: Callback`
    `dtype: raw`
    `default: '0'` 
if you put quotes around the variable it returns this 
`self.xmlrpc_client_0."set_rmt_freq"(self.freq)`
 ` ^`
`SyntaxError: invalid syntax`

Reply:
-   id: callback
    label: Callback
    dtype: string
    default: '0'
    results in the same error
    `self.xmlrpc_client_0.set_'rmt_freq'(self.freq)`
`                               ^`
`SyntaxError: invalid syntax`

Reply:
Yes, the quotes can be removed by taking the substring: `str(callback)[1:-1]`, this is done several in other templates. It's a little ugly, and I'll post a PR later today which enables us to do it in a more robust and elegant way.

Reply:
But you're saying that the raw type is being treated differently on master than on the older branches, yes?

Reply:
Yes, caused by https://github.com/gnuradio/gnuradio/pull/4629